notifier/selfstate/heartbeat/notifier.go

```diff
@@ -3,29 +3,42 @@ package heartbeat
 import (
 	"fmt"
 
+	"github.com/moira-alert/moira/metrics"
+
 	"github.com/moira-alert/moira"
 )
 
 type notifier struct {
-	db  moira.Database
-	log moira.Logger
+	db              moira.Database
+	log             moira.Logger
+	notifierIsAlive metrics.Meter
 }
 
-func GetNotifier(logger moira.Logger, database moira.Database) Heartbeater {
+func GetNotifier(logger moira.Logger, database moira.Database, notifierIsAlive metrics.Meter) Heartbeater {
 	return &notifier{
-		db:  database,
-		log: logger,
+		db:              database,
+		log:             logger,
+		notifierIsAlive: notifierIsAlive,
 	}
 }
 
 func (check notifier) Check(int64) (int64, bool, error) {
-	if state, _ := check.db.GetNotifierState(); state != moira.SelfStateOK {
+	state, _ := check.db.GetNotifierState()
```

Comment:
Why are we ignoring the error from the database?

Reply:
<img width="335" alt="image" src="https://github.com/moira-alert/moira/assets/15986169/0ef1c9a9-101c-4914-845e-18a25a98de06">

Reply:
Ok, I get it